tools/cldr/build.xml

```diff
@@ -0,0 +1,71 @@
+<!-- © 2020 and later: Unicode, Inc. and others.
+     License & terms of use: http://www.unicode.org/copyright.html -->
+
+<!-- FIXME: more instructions here. -->
+
+<!-- Using the directory of this build.xml file as ${basedir}. -->
+<project name="CLDR" default="install" basedir=".">
+
+    <!-- Installs prerequisite data from CLDR (test data and CLDR libraries).
+         This target will not rebuild the ICU data however (since that's a
+         lengthy process). -->
+    <target name="install" depends="clean, copy-cldr-testdata, install-cldr-libs"/>
+
+    <!-- Cleans CLDR dependencies (testdata, CLDR libraries). This target does
+         not delete any generated ICU data. -->
+    <target name="clean" depends="clean-cldr-testdata"/>
+
+    <!-- Initialize any properties not already set on the command line. -->
+    <target name="init-args">
+        <property environment="env"/>
+        <!-- Inherit properties from environment variable unless specified. As usual
+             with Ant, this is messier than it should be. All we are saying here is:
+             "Use the property if explicitly set, otherwise use the environment variable."
+             We cannot just set the property to the environment variable, since expansion
+             fails for non existant properties, and you are left with a literal value of
+             "${env.CLDR_DIR}". -->
+        <condition property="cldrDir" value="${env.CLDR_DIR}">
+            <isset property="env.CLDR_DIR"/>
+        </condition>
+        <fail unless="cldrDir"
+             message="Set the CLDR_DIR environment variable (or cldrDir property) to the top level CLDR source directory (containing 'common')."/>
+
+        <!-- The directory for CLDR testdata (this has a reasonable default value). -->
+        <property name="testDataDir" value="${basedir}/../../testdata/cldr"/>
```

Comment:
For now, let's make this an ICU4C testdata directory:

        <property name="testDataDir" value="${basedir}/../../icu4c/source/test/testdata/cldr"/>

I will propose a follow-up change to also copy it to ICU4J, as @younies is currently working on reading these files for Java testing purposes. After this release, we (I) can make a concerted effort to unify on a shared testdata dir, but that has too many consequences at this time.

Reply:
FYI: from curiosity and github's suggestion, "Add more commits by pushing to the build_stuff branch on hagbard/icu.", I experimented to see if I have the rights to do so. The answer seems to be no, and the error is as follows:

```
$ git remote -v | grep hagbard
hagbard	https://github.com/hagbard/icu.git (fetch)
hagbard	git@github.com:hagbard/icu.git (push)
$ git tree -3
* 6543ac9947 (HEAD -> build_stuff) ICU-21066 Move CLDR testDataDir to an ICU4C location.
* 7ee001c848 (hagbard/build_stuff) ICU-21066 Improvements in Ant build files
* 0eed48038b ICU-20725 Fix stack overflow of u_unescapeAt
$ git push
ERROR: Authentication error: Authentication required: You must have push access to verify locks
error: failed to push some refs to 'github.com:hagbard/icu.git'
```

Reply:
I probably have it locked down.

Reply:
We should copy the files into two places:

- `${basedir}/../../icu4c/source/test/testdata/cldr`
- `${basedir}/../../icu4j/main/tests/core/src/com/ibm/icu/dev/data/cldr`

Reply:
Ahh, if that's the case, then I'll remove this (replace with a TODO) to unblock things.